custom/icds/location_reassignment/forms.py

```diff
@@ -29,11 +29,11 @@ class LocationReassignmentRequestForm(BulkUploadForm):
                                f"(Only for {SPLIT_OPERATION} and {EXTRACT_OPERATION})"))]
     action_type = forms.ChoiceField(choices=ACTION_TYPE_CHOICES,
                                     initial=VALIDATE,
-                                    widget=SelectToggle(choices=ACTION_TYPE_CHOICES))
+                                    )
 
     def crispy_form_fields(self, context):
         crispy_form_fields = super(LocationReassignmentRequestForm, self).crispy_form_fields(context)
         crispy_form_fields.extend([
-            crispy.Div(InlineField('action_type'), css_class="col-sm-6"),
+            crispy.Div(InlineField('action_type'), css_class="col-sm-4"),
```

Comment:
Adding `hqwebapp-select2` here and `hqwebapp/js/widgets` to the js file should make this a select2, which would be good for consistency.

Reply:
I added that and then to the js file which made the dropdown stretch to full screen which didn't look right so not making this change for now but happy to take as a follow up if you have suggestions on how to set that right.